target/linux/ipq40xx/image/Makefile

```diff
@@ -249,6 +249,22 @@ define Device/compex_wpj428
 endef
 TARGET_DEVICES += compex_wpj428
 
+define Device/corewav_m018u0-s1
+	DEVICE_VENDOR := Corewav
+	DEVICE_MODEL := M018U0
+	DEVICE_VARIANT := S1
+	BOARD_NAME := m018u0-s1
```

Comment:
What's that for?

Reply:
I just refer to other people's writing (eg. dk01.1-c1, qxwlan_e2600ac-c1)

Reply:
Well, then remove it and check whether it's still working.

Reply:
I removed "BOARD_NAME" and just found #2790 .